pkgs/tools/misc/opentelemetry-collector/contrib.nix

```diff
@@ -0,0 +1,43 @@
+{ buildGoModule
+, fetchFromGitHub
+, lib
+}:
+
+buildGoModule rec {
+  pname = "opentelemetry-collector-contrib";
+  version = "0.43.0";
+
+  src = fetchFromGitHub {
+    owner = "open-telemetry";
+    repo = "opentelemetry-collector-contrib";
+    rev = "v${version}";
+    sha256 = "sha256-ktzP+ugG2sa0v8B1Zp47o8Bmpxv98zQyFyWf9QfQRoQ=";
+  };
+  vendorSha256 = "sha256-E4P2aNE31ATYjvEOnAOsbIRLInfROUc7Gv1eIrjXVZM=";
```

Comment:
Need to use `proxyVendor` here like in the original derivation and update the `vendorSha256` to fix the darwin/linux hash mismatch.

Reply:
ah, ok I'll add it back with a note